.github/workflows/call-backport-label-updater.yaml

```diff
@@ -0,0 +1,19 @@
+---
+    name: Call Backport Label Updater
+    on:
+      pull_request:
+        types:
+          - closed
+        branches:
+          - v1.14
+    jobs:
+      call-backport-label-updater:
+        if: |
+            github.event.pull_request.merged == true &&
+            contains(github.event.pull_request.title, 'Backports') &&
```

Comment:
Maybe we should have this instead
```
contains(github.event.pull_request.body, 'upstream-prs')
```

which will work for all cases that we want automation.

Reply:
Fixed. I've updated also https://github.com/cilium/cilium/pull/27877 and https://github.com/cilium/cilium/pull/27879